priv/repo/migrations/20160913080731_update_team_table.exs

```diff
@@ -0,0 +1,12 @@
+defmodule EspiDni.Repo.Migrations.UpdateTeamTable do
+  use Ecto.Migration
+
+  def change do
+    rename table(:teams), :token, to: :slack_token
+
+    alter table(:teams) do
+      add :google_token, :string
+      add :google_refresh_token, :string
+    end
+  end
```

Comment:
It's at such early stages loosing the data is no big deal, but yeah, I should probably do it the right way. Will split them.